examples/test/vasp_param_from_incar.json

```diff
@@ -37,7 +37,6 @@
     "_comment": "pert xz to work around vasp bug...",
     "pert_xz": 0.01,
     "max_miller": 2,
-    "static-opt": false,
     "relax_box": false,
     "_comment": "06.phonon",
     "supercell_matrix": [
```

Comment:
These examples are for the old autotest. The input examples for the current autotest are already in doc. We can considering replacing these old examples (maybe in another pr).

Reply:
OK. Looking forward to it.